classes/privacy/provider.php

```diff
@@ -0,0 +1,54 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Privacy Subsystem implementation for block_course_summary.
+ *
+ * @package    block_course_template
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace block_course_template\privacy;
+
+defined('MOODLE_INTERNAL') || die();
+
+/**
+ * Privacy Subsystem for block_course_summary implementing null_provider.
+ *
+ * @copyright 	Catalyst IT {@link http://catalyst.net.nz}
+ * @author 	bO Pierce 
+ * @contributor Michael Nixon <michael.nixon@catalyst.net.nz>
```

Comment:
sorry - being picky again.... we are using PHPdocs here - @contributor is not a valid tag and will throw an error on phpdoc validation. - just remove it :-)

Travis-ci hasn't been always able to perform the moodle phpdoc check - there's a new version of the travis integration that does but I haven't used it yet myself!